ex/ThirdPartyDemos/martin-doyle/05 - Http Daemon ORM/source/Project05HttpDaemon.dpr

```diff
@@ -0,0 +1,102 @@
+program Project05HttpDaemon;
```

Comment:
Note that the daemons support is about to be highly enhanced and facilitated.

Reply:
You will make this even easier to use :)